.gitignore

```diff
@@ -25,6 +25,7 @@ workers/unity/*.sln
 *.svd
 *.DotSettings.user
 _ReSharper.Caches
+*.DotSettings
```

Comment:
We use these to keep code styles in sync

Reply:
Yeah this isn't an acceptable solution imo